doc/api-extensions.md

```diff
@@ -211,3 +211,6 @@ Implements the X-LXD-write header which can be one of "overwrite" or "append".
 
 ## network\_dhcp\_expiry
 Introduces "ipv4.dhcp.expiry" and "ipv6.dhcp.expiry" allowing to set the DHCP lease expiry time.
+
+## storage\_lvm\_vg\_rename
```

Comment:
Not really sure if this counts as an api extension as it doesn't really introduce a new configuration key. I added it for now but I can simply drop that commit.